src/components/App.styles.ts

```diff
@@ -0,0 +1,15 @@
+import { createUseStyles } from "react-jss";
+
+export default createUseStyles({
+  root: {},
+  recordContainer: (props) => ({
+    alignSelf: "center",
+    alignItems: "center",
+    display: "flex",
+    flexDirection:
+      props.isMobile && props.orientation === "portrait" ? "column" : "row",
+    justifyContent: "space-around",
+    padding: "0.5rem",
+    width: "100%",
+  }),
+});
```

Comment:
This could on the default global styles or in the Tabs component.

Reply:
further review, i dont think we can put it in default global styles. there are some props I'm utilizing to control `flexDirection`